<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debug</class>
 <widget class="QWidget" name="Debug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="sensor_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Video Source : Sensor.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>MAIN MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tpg_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Video Source : TPG.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="isp_en_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable ISP Module   : ISP.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sd_save_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save the image in the SD : SD</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QPushButton" name="tpg_back_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tpg_cfg_btn">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>320</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>configure</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_H_RAMP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_V_RAMP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_TEMPORAL_RAMP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_SOLID_RED</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_SOLID_GREEN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_SOLID_BLUE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_SOLID_BLACK</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_SOLID_WHITE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_COLOR_BARS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_10">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_ZONE_PLATE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_TARTAN_COLOR_BARS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_12">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_CROSS_HATCH</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_13">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_RAINBOW_COLOR</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_14">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_HV_RAMP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_15">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_CHECKER_BOARD</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_16">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_PBRS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_17">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>90</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_DP_COLOR_RAMP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_18">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>140</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_DP_BW_VERTICAL_LINE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_19">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>180</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_DP_COLOR_SQUARE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_20">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>XTPG_BKGND_LAST</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>é¡µ</string>
    </attribute>
    <widget class="QPushButton" name="isp_cfg_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>300</y>
       <width>92</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>configure</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="isp_back_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ISP_EN</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TPG_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>TPG_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="DPC_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>DPC_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="BLC_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>BLC_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="LSC_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>LSC_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="NR_RAW_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>NR_RAW_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="AWB_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>AWB_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="WBC_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>WBC_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="STA_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>STA_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="GB_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GB_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="DMS_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>DMS_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CCM_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CCM_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="GAMMA_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>110</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GAMMA_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CSC_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>150</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CSC_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="NR_YUV_MODULE_EN_check">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>NR_YUV_MODULE_EN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>æ­¤å¤å¾éä¸å¦æ¯æ¯å¦åè½¬çææï¼ä»ç¨ä½æµè¯!!!</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
